catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java

```diff
@@ -163,8 +163,7 @@ public void run() {
                         Exceptions.getFullMessage(e),
                         e);
 
-                processingDetails.add(new ProcessingDetailsImpl(sourceId,
-                        new Exception(Exceptions.getFullMessage(e))));
+                processingDetails.add(new ProcessingDetailsImpl(sourceId, e));
```

Comment:
If we take `new Exception(Exceptions.getFullMessage(e))` out, we won't get the list of all messages in the `processingDetails` object. Not sure if this is good or bad. Probably need to check with @stustison or @pklinef. Same for the other similar change in `CatalogFrameworkImpl`.